package.json

```diff
@@ -218,8 +216,10 @@
     "lint:mixedindent": "mixedindentlint --ignore-comments \"client/**/*.scss\" \"assets/**/*.scss\" \"**/*.js\" \"**/*.jsx\" \"!build/**\" \"!node_modules/**\" \"!public/**\" \"!client/config/index.js\"",
     "precommit": "npm run -s install-if-no-packages && node bin/pre-commit-hook.js",
     "prepush": "npm run -s install-if-no-packages && node bin/pre-push-hook.js",
-    "rm": "node bin/rm.js",
-    "prestart": "npm run -s install-if-deps-outdated && node bin/welcome.js && node bin/version-check.js",
+	"rm": "node bin/rm.js",
+	"prestart:check-version": "npx check-node-version --node \"^10.5\" --npm \"~6\"",
```

Comment:
I'd prefer to use `check-node-version --package` and put the `^10.5` and `~6` in the `engines` section of `package.json`. What do you think about that? We already use the `engines` property and `.nvmrc`, I'd rather we don't have to remember to update the version in 3 places on every update.

If not, what about just removing the `engines` property?

Reply:
using --package makes sense!